sections/image-banner.liquid

```diff
@@ -271,6 +335,26 @@
           "id": "text",
           "default": "Give customers details about the banner image(s) or content on the template.",
           "label": "t:sections.image-banner.blocks.text.settings.text.label"
+        },
+        {
+          "type": "select",
+          "id": "text_style",
+          "options": [
+            {
+              "value": "body",
+              "label": "t:sections.image-banner.blocks.text.settings.text_style.options__1.label"
+            },
+            {
+              "value": "subtitle",
+              "label": "t:sections.image-banner.blocks.text.settings.text_style.options__2.label"
+            },
+            {
+              "value": "uppercase",
```

Comment:
@wiktoriaswiecicka @melissaperreault What are your thoughts on adding a new block here since it was added on image with text too? Since translations are already back maybe we can create a follow up to address that if you think it makes sense to add it as well :)

Reply:
I planned to revisit later on, but if we have a bandwidth to do it - let's add it please! 🙂 I'll make sure to add it to the list of features for the future sections with the same block layout (rich text etc). @sofiamatulis I just tagged you on the updated settings on Figma.

Reply:
I have just added it to the new issue I created since well need translations :)

https://github.com/Shopify/dawn/issues/959